spec/models/spree/calculator/price_sack_spec.rb

```diff
@@ -49,6 +49,31 @@
     end
   end
 
+  context "minimal amount is float" do
+    before do
+      calculator.preferred_minimal_amount = 16.5
+      calculator.preferred_normal_amount = 5
+      calculator.preferred_discount_amount = 1
+    end
+
+    context "with price bellow minimal amount" do
+      let(:price) { 8 }
+
+      it "returns the correct value of cost" do
+        expect(calculator.compute(line_item)).to eq(5)
+      end
+    end
+
+    context "with price above minimal amount" do
+      let(:price) { 8.5 }
```

Comment:
sorry, is this correct? In line 54 I see `preferred_minimal_amount = 16.5` so it seems that the code contradicts the `context` description. Am I missing something?

Reply:
ah, yes, you are missing quantity = 2
So 8.5 * 2 = 17 which is above 16.5
it's an existing readability problem in the spec.

Reply:
@sauloperez @luisramos0 It's true, thanks for the alert, I didn't notice this. I will change the price to 8.25, which gives 8.25 * 2 = 16.5. But the context is for above the minimal amount (16.5), which 17 is, doesn't sound correct?

Reply:
I dont think you need to change the value, you could maybe make the quantity = 2 more obvious in some way.

Reply:
@luisramos0 Something like this?
```ruby
  context "minimal amount is float" do
    before do
      calculator.preferred_minimal_amount = 16.5
      calculator.preferred_normal_amount = 5
      calculator.preferred_discount_amount = 1
      line_item.quantity = 2
    end
```

Reply:
Yes, I think it's fine. Although the assignment has no effect, next time I read this I'll know the dev was trying to tell me I should know the quantity to understand the test :ok_hand:

Reply:
Yeah, I agree. Great advice @sauloperez, I'll keep in mind.

Reply:
I made an amend commit with this change, not worth another commit for this, alright?